doc/intergrations/intergration-with-datadog.md

```diff
@@ -0,0 +1,81 @@
+Datadog provides easy integration of log and metrics collection in the kubernetes environment.
+This document describes how to integrate the Datadog with Starrocks.
+
+> see https://docs.datadoghq.com/containers/kubernetes/installation/?tab=helm for more details.
+
+## 1. Install the Datadog chart
+
+If this is a fresh install, add the Helm Datadog repo:
+
+```bash
+helm repo add datadog https://helm.datadoghq.com
+helm repo update
+```
+
+Create a Kubernetes Secret to store your Datadog API key and app key:
+
+```bash
+export DD_API_KEY=xxx
+export DD_APP_KEY=yyy
+kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY --from-literal app-key=$DD_APP_KEY
+```
+
+Set the following parameters in your `datadog-values.yaml` to reference the secret:
+
+```bash
+datadog:
+  apiKeyExistingSecret: datadog-secret
+  appKeyExistingSecret: datadog-secret
```

Comment:
duplicated line

Reply:
one is for apiKey, another is for appKey. see  https://docs.datadoghq.com/containers/kubernetes/installation/?tab=helm

Reply:
ok. -_-!